### ISSUE TITLE
Typescript Types

### ISSUE THREAD

**Issue by CMCDragonkai** [toolbuilder/await-for-it#1]
Do you have plans to include TS types here?

**Comment by justin0mcateer** [comment]
I am very interested in this. I've written types for a fairly good portion of the library. Would you be willing to accept a Pull Request?

**Comment by toolbuilder** [comment]
Yes, a pull request would be great. I wasn't having much success. 

    On Friday, February 17, 2023 at 06:25:39 PM EST, justin0mcateer ***@***.***> wrote:  
 
 


I am very interested in this. I've written types for a fairly good portion of the library. Would you be willing to accept a Pull Request?
